local_adaptation/README.md

```diff
@@ -0,0 +1,28 @@
+##  Local Adaptation NWF
+
+This is a simulation in which local adaptation is achieved through QTL mutations arising in a 20kb region of the 1e6bp genomic element. Neutral mutations may be overlayed for downstream analysis (say, using msprime), as they are excluded from this example. 
+
+`LAimg_greyscale.png` is the example greyscale image used in the current script, but this can be easily exchanged for another greyscale image. If you choose to upload an RBG image, make sure to edit `mapImage.floatK` in line 61 to `mapImage.floatG`. Greyscale is recommended. 
+
+An interesting aspect to this script is how we approach fitness scaling. Notice that we first measure local density, like in other examples, but then we also construct phenotypes and fitness effects from QTLs. The phenotypes of individuals are calculated as the sum of the `m2` mutations (QTL). So then, we have these two elements of fitness scaling: 
+
+first in line 76:
+```
+inds.fitnessScaling = 1 / (1 + RHO * competition);
+```
+
+second in line 87:
+```
+inds.fitnessScaling =   inds.fitnessScaling * qtl_fit;
+```
+where qtl_fit is defined as:
+```
+qtl_fit = 1 + dnorm(phenotype, optimum, SK);
+qtl_fit = qtl_fit/max(qtl_fit);
```

Comment:
Good point @bhaller! 
What about? 

```suggestion
    qtl_fit = dnorm(phenotype, optimum, SK);
    qtl_opt = dnorm(0.0, 0.0, SK);
    qtl_fit = qtl_fit/qtl_opt;
```
This was the main part I helped with, but I didn't have a good rationale for rescaling based on min distance to optimum during each eval -- just wanted competition based fitness to be comparable in scale to qtl fitness. This should rescale by global max fitness. Simulation behavior looks good with this change.

Reply:
@silastittes Yes, this is the more typical approach; scale the height by the height at the optimum, so that the value for an individual at the optimum is 1.0.  You then have two free parameters: `final_fit = a + b * qtl_fit`, if you see what I mean.  In other words, you can stretch the fitness function vertically (b), and you can shift it vertically without stretching it (a), both of which would affect the calculated fitness values.  Your code implicitly has a=0 and b=1, I think; that's fine, if that works for your purposes.

Reply:
That makes sense. Thanks for walking through it!